sdcm/tester.py

```diff
@@ -906,6 +907,12 @@ def prepare_kms_host(self) -> None:
             'key_provider': 'KmsKeyProviderFactory',
             'kms_host': kms_host,
         }
+
+        if SkipPerIssues("scylladb/scylla-enterprise#4246", self.params):
```

Comment:
@mykaul 

as long as this issue would be open, we'll be disabling file streaming, on all places we by default enabled KMS